pandas/_libs/src/inference.pyx

```diff
@@ -222,14 +222,16 @@ cdef _try_infer_map(v):
     return None
 
 
-def infer_dtype(object value):
+def infer_dtype(object value, bint skipna=False):
     """
     Effeciently infer the type of a passed val, or list-like
     array of values. Return a string describing the type.
 
     Parameters
     ----------
     value : scalar, list, ndarray, or pandas type
+    skipna : bool
```

Comment:
@cpcloud : Thoughts about my comment regarding the eventual change of default value?